cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -212,14 +226,380 @@ data ProtocolParameters =
        --
        -- This is the \"tau\" incentives parameter from the design document.
        --
-       protocolParamTreasuryCut :: Rational
+       protocolParamTreasuryCut :: Rational,
+
+       -- Introduced in Alonzo
+
+       -- | Cost in ada per byte of UTxO storage (instead of
+       --protocolParamMinUTxOValue in the Alonzo era onwards).
+       protocolParamUTxOCostPerByte :: Maybe Lovelace,
+
+       -- | Cost models for non-native script languages.
+       protocolParamCostModels :: Maybe CostModel,
+
+       -- | Prices of execution units (for non-native script languages).
+       protocolParamPrices :: Maybe Prices,
```

Comment:
Naming! :smile: 

We should probably give this a more specific name "prices" is a bit too generic.

I'd suggest:`Prices` to `ExecutionUnitPrices`

This will also need to be per-language in the end, once we have many plutus versions. So I suggest: `Map AnyScriptLanguage ExecutionUnitPrices`.